package.json

```diff
@@ -28,5 +28,9 @@
   },
   "lint-staged": {
     "*.json": "prettier --write"
+  },
```

Comment:
looks like a mistake

please run the following then commit:
`npm uni pretty run`